packages/subapp-web/lib/start.js

```diff
@@ -108,6 +127,20 @@ module.exports = function setup() {
                 },
                 { concurrency }
               );
+              if (xarcSSREmitter) {
+                xarcSSREmitter.emit("web_ssr", {
+                  action: `render-group`,
+                  group,
+                  labels: ["group"],
+                  duration: Date.now() - startTimes[`render-grp-${group}`]
+                });
+                xarcSSREmitter.emit("web_ssr", {
```

Comment:
what's the context behind emitting two events with the same time?  please add comment to explain that.

Please use the same now timestamp for both.

Reply:
`render-group` event is to capture the duration for just rendering of groups, whereas `load-group` captures the total processing time for that group. I am capturing the start times of the events accordingly. I was a little skeptical about the `load-group` event. But thought, its nice to have one events for the total time. Please let me know your suggestions here.

Reply:
so `load-group` is the total time of prepare + render?
that's fine, but maybe a better name?  

name suggestions:

`group-total-${group}`
`group-render-${group}`